nodejs/lib/util/defaults.js

```diff
@@ -9,7 +9,8 @@ var Defaults = {
 	baseTransportOrder: ['comet', 'web_socket'],
 	transportPreferenceOrder: ['comet', 'web_socket'],
 	upgradeTransports: ['web_socket'],
-	restAgentOptions: {maxSockets: 40, keepAlive: true}
+	restAgentOptions: {maxSockets: 40, keepAlive: true},
+	agent: 'nodejs/' + process.version
```

Comment:
We need to document these agent identifiers as we add them. I've started a document (internal for now) so we can pull this information together, which you can find [here](https://docs.google.com/document/d/1duUdRknI1QNkanASgkTYo19LGXQIeT8oFlXS7TCU-Ek/edit?usp=sharing).

Also, looking at the docs for [process.version](https://nodejs.org/api/process.html#process_process_version) it appears that you'll be emitting a string that has a 'v' prefix. That would be superfluous and probably technically wrong if we decide (as I think could be the case) that this second component of each product added to the agent should conform to something sensible like [SemVer](https://semver.org/). This needs stronger specification and I would like to at least lay that down in the document I referenced earlier in this comment.

Reply:
I've resolved this now by using `platform.versions.node` which is valid SemVer.